app/src/main/java/in/avimarine/boatangels/db/objects/User.java

```diff
@@ -0,0 +1,26 @@
+package in.avimarine.boatangels.db.objects;
+
+/**
+ * Created by Motim on 12/25/2017.
+ */
+
+public class User extends BaseDbObject {
+
+  public String DisplayName;
+  public String Mail;
+  public String Phone;
+  public String Country;
+  public String FirstJoinTime;
```

Comment:
This should be Date or if the DB does not accept date objects than Long. when using string for date it might cause problems with deserializing the String

Reply:
User object also missed uid field

Reply:
change type of "firstjointime" from String to Date  (its save in DB under 'timestamp' type ).

Reply:
I see there is "uid" in object !